<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformationWidget</class>
 <widget class="QWidget" name="InformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="informationGroupBox">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="producer">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Producer:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="producerValue">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="publicationDate">
            <property name="text">
             <string>Publication date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="publicationDateValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="copyright">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Copyright:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="copyrightValue">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="description">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="descriptionValue"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>publicationDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>publicationDateValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publicationDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>InformationWidget</receiver>
   <slot>refreshInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publicationDateValue</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>InformationWidget</receiver>
   <slot>refreshInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>producerValue</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>InformationWidget</receiver>
   <slot>refreshInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyrightValue</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>InformationWidget</receiver>
   <slot>refreshInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionValue</sender>
   <signal>textChanged()</signal>
   <receiver>InformationWidget</receiver>
   <slot>refreshInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
